npm_and_yarn/lib/dependabot/npm_and_yarn/file_updater/npm_lockfile_updater.rb

```diff
@@ -467,9 +467,9 @@ def post_process_npm_lockfile(lockfile_content)
           git_dependencies_to_lock.each do |_, details|
             next unless details[:from]
 
-            new_r = /"from": "#{Regexp.quote(details[:from])}#[^\"]+"/
-            old_r = %("from": "#{details[:from]}")
-            updated_content = updated_content.gsub(new_r, old_r)
+            locked_from = %("from": "#{details[:version]}")
```

Comment:
This makes a tonne of sense, but only after I thought about it for ages. Can you add a one-line comment saying `We set the requirement to be the version, so that's what the new from will be`?

Reply:
Yes good shout!